### DISCUSSION TITLE
Question on Species selection /request

### DISCUSSION THREAD

**Discussion by Solas67** [bcholmes/StarTrek2d20#241] [Q&A]
I've been tinkering with the creator and it looks and works very well, though I am noticing the Tribble Player Character doesn't seem to activate under any setting, is that perhaps not enabled yet? 

Would it be possible to add the Horta as a species also?

**Comment by bcholmes** [comment]
Horta were introduced in Captain's Log, but I've now included some of the Continuing Missions talents as well. 

The Tribbles should be selectable in the latest version.

**Comment by Solas67** [comment]
Thank you so much for the quick feedback. Greatly appreciated and loving
your work, I'm super happy Modiphus link directly to you as a source.
